components/event-gateway/integration_test/nats_auth_test.go

```diff
@@ -36,19 +37,25 @@ func (s *Suite) initNATSAuthTest() error {
 		return err
 	}
 
-	policiesResp, err := s.AuthzClient.ListPolicies(ctx, &authz.ListPoliciesReq{})
+	policiesResp, err := s.AuthzClient.ListPolicies(ctx, &policies.ListPoliciesReq{})
 	if err != nil {
 		return err
 	}
 
-	for _, p := range policiesResp.Policies {
-		if p.Resource == "ingest:*" {
-			s.AuthPoliciesToRestore = append(s.AuthPoliciesToRestore, p)
+PolicyLoop:
```

Comment:
The comment says this test `initNATSAuthTest deletes a built-in authorization policy that allows all tokens permission to the ingest endpoints/actions.` That's just the old default policy right? I think we should be able to rewrite this test assuming we're on v2 with no default policies. Or else let's just create a one-off policy that denies `ingest:*` to the test token